pnpm-workspace.yaml

```diff
@@ -6,7 +6,7 @@ packages:
   # Intentionally exclude the `npm` `turbo` package from the workspaces.
   - "!packages/turbo"
   - "packages/turbo-tracing-next-plugin/test/with-mongodb-mongoose"
-  - "crates/turbopack/tests/node-file-trace"
   - "crates/turbopack/tests/snapshot"
   - "crates/next-dev/tests"
   - "crates/*/js"
+  - "!crates/turbopack/tests/node-file-trace"
```

Comment:
Do you need to explicitly remove this? There isn't anything that adds this directory with a glob